starter_kits/Ruby/hlt/direction.rb

```diff
@@ -0,0 +1,57 @@
+require 'commands'
+
+# Holds positional arrays in relation to cardinal directions
+module Direction
+  NORTH = [0, -1]
+  SOUTH = [0, 1]
+  EAST = [1, 0]
+  WEST = [-1, 0]
+
+  STILL = [0, 0]
+
+  # Returns all contained items in each cardinal
+  # :return: An array of cardinals
+  def self.all_cardinals
+    return [NORTH, SOUTH, EAST, WEST]
+  end
+
+  # Converts from this direction tuple notation to the engine's string notation
+  # :param direction: the direction in this notation
```

Comment:
`non-blocking`
Nitpick: These comments are yanked from the python version, and not of proper Ruby YARD doc style. I'll open an issue to resolve once this is merged.